### ISSUE TITLE
Hydra strict mode

### ISSUE THREAD

**Issue by omry** [asherp/hourly#2]
I see you are not using strict mode here:

https://github.com/asherp/hourly/blob/22cd2d48559c849d057bb2a904f61db451808ec6/cli/main.py#L396

if I had to guess, you found it conflicts with 
`cfg.report.timesheet`.

Strict mode is highly recommended.
There are a few ways to change that value and keep strict mode on.

1. put 
```yaml
report:
  timesheet = '???'
```
in your config.
This would be a place holder for timesheet that must be populated when used.

2. open up the config for changes when modifying it.
see [open_dict in OmegaConf](https://omegaconf.readthedocs.io/en/1.4_branch/usage.html#struct-flag).

**Comment by asherp** [comment]
Thanks, Omry! I had been going back and forth with whether or not to use strict mode.

**Comment by omry** [comment]
Strongly suggest that you do.

To address another common pattern without strict mode:

```python
if cfg.foo is not None:
```

Can be replaced with this in strict mode:
```python
if "foo" in cfg:
```

Let me know if you have other issues with strict mode.

**Comment by omry** [comment]
By the way, you are very welcome to join the Hydra chat (https://hydra-framework.zulipchat.com/)

**Comment by asherp** [comment]
Closing

**Comment by asherp** [comment]
This was resolved in the latest release 0.3.9
